governance/program/src/state/enums.rs

```diff
@@ -152,6 +154,15 @@ pub enum VoteThreshold {
     /// Once the quorum is achieved a simple majority (50%+1) of Yes votes is required for the vote to succeed
     /// Note: Quorum is not implemented in the current version
     QuorumPercentage(u8),
+
+    /// Disabled vote threshold indicates the given voting population (community or council) is not allowed to vote
+    /// on proposals for the given Governance
+    Disabled,
```

Comment:
I'm not sure how sensitive you are to account layout on the frontend, but there might be an issue since this is only size 1 while the others are size 2.  When borsh serializes, it'll pack it in to 1 byte.  It may be worth adding a dummy `u8` value.

Reply:
It needs a custom serialisation logic and I accounted for that in the client package https://github.com/solana-labs/oyster/blob/fe749f0190f54b5b3a4c6ed5decf6100dfb58a9c/packages/governance-sdk/src/governance/serialisation.ts#L159